research/auto-fed-rl/requirements.txt

```diff
@@ -0,0 +1,4 @@
+nvflare>=2.3.0
```

Comment:
we can remove nvflare from this req, it will be covered in the general req

Reply:
See here for example https://github.com/NVIDIA/NVFlare/blob/dev/examples/hello-world/hello-numpy-sag/requirements.txt

Reply:
At least in research, we should follow the template directory structure https://github.com/NVIDIA/NVFlare/tree/dev/research#requirements

Reply:
yeah, fed-sm still has it